go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/containerd/nerdctl
 
 go 1.19
 
+// 82ec62b86582 refers to the feature branch of https://github.com/containerd/containerd/pull/8334.
+// It's used to make the e2e tests pass.
+// TODO: Before merging this PR, we need to remove this `replace` directive and use the released containerd version.
+replace github.com/containerd/containerd => github.com/davidhsingyuchen/containerd v0.0.0-20230331013921-82ec62b86582
```

Comment:
Why ?

Reply:
Without this temp change, https://github.com/containerd/containerd/issues/8326 makes `IO.Wait()` hang forever even if `IO.Cancel()` has been invoked.